packages/tasks/src/tasksData.ts

```diff
@@ -54,6 +54,7 @@ export const TASKS_DATA: Record<PipelineType, TaskData | undefined> = {
 	"image-segmentation": getData("image-segmentation", imageSegmentation),
 	"image-to-image": getData("image-to-image", imageToImage),
 	"image-to-text": getData("image-to-text", imageToText),
+	"image-to-video": undefined,
```

Comment:
Just a question to help me understand how it works, would creating an entry in `tasks/data` and populating it here would be enough to have the task listed in https://huggingface.co/tasks ?

Reply:
Awesome, thanks!